spring/boot3-autoconfigure/src/test/resources/config/application-autoConfTest.yml

```diff
@@ -4,6 +4,12 @@ management:
       prometheus:
         enabled: true # In order to inject PrometheusMeterRegistry
 
+  # Spring Boot 3 has changed the property name for metrics.
```

Comment:
Maybe remove this comment from the .yml files for Spring Boot 3? This is useful for migration but these files were done with migration 😉